src/conversions.c

```diff
@@ -58,7 +58,7 @@ to_float(PyArray_Descr *descr,
 }
 
 
-/* TODO: Used only for the type inference (which glances over the bugs though) */
+/* Note: Currently used in the iteger code as a fallback */
```

Comment:
Still need that one in the integer code :/ (otherwise, have to hold on to a `np.dtype("double")`.  That is probably not terrible, but for another time maybe...